extensions/amp-nexxtv-player/0.1/validator-amp-nexxtv-player.protoascii

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-nexxtv-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-nexxtv-player extension .js script"
+  satisfies: "amp-nexxtv-player extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
```

Comment:
Please make this `unique: true`. Warning is for older scripts that were grandfathered in before it was standard for this to be an error.